frontend/src/views/Project/CertificatesPage/components/CertificatesTab/components/CertificateModal.tsx

```diff
@@ -14,14 +14,21 @@ import {
   SelectItem
 } from "@app/components/v2";
 import { useWorkspace } from "@app/context";
-import { CaStatus, useCreateCertificate, useGetCert, useListWorkspaceCas } from "@app/hooks/api";
+import {
```

Comment:
one thing I noticed here is that the Certificate Collection remains blank in the "view certificate" modal even when it actually belongs to a collection
<img width="1202" alt="image" src="https://github.com/user-attachments/assets/564ac0f0-fb63-44ed-8adb-f0c899553179">

Reply:
Removed Certificate Collection field for when viewing the details of an existing certificate because it can actually belong to multiple collections and isn't an inherent property of the certificate itself.